protocol/x/clob/keeper/liquidations.go

```diff
@@ -109,8 +123,14 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 			unfilledLiquidations = append(unfilledLiquidations, *liquidationOrder)
 		}
 	}
+	telemetry.MeasureSince(
+		startPlaceLiquidationOrders,
+		types.ModuleName,
+		metrics.ClobLiquidateSubaccounts_PlaceLiquidations,
+	)
 
 	// For each unfilled liquidation, attempt to deleverage the subaccount.
+	startDeleverageSubaccounts := time.Now()
```

Comment:
LMK if anyone can think of a cleaner way to do this. Could potentially abstract these loops into functions so a `defer` statement could be used to measure latency.

Reply:
yeah I think breaking this down to individual functions would be cleaner since this function is kind of long. 

no need to block on this though, we can do that separately

Reply:
Agreed, created an M6 ticket [here](https://linear.app/dydx/issue/CLOB-963/break-up-liquidatesubaccountsagainstorderbook-steps-into-functions). LMK if you think this should be M5